scala/private/rules/scala_library.bzl

```diff
@@ -161,6 +161,8 @@ _scala_library_for_plugin_bootstrapping_attrs.update(
     common_attrs_for_plugin_bootstrapping,
 )
 
+_scala_library_for_plugin_bootstrapping_attrs["build_ijar"] = attr.bool(default = True)
```

Comment:
WDYT about changing this to a dict? To follow the existing pattern https://github.com/bazelbuild/rules_scala/blob/6ba125f449633332766b80d62fd47393589dd73f/scala/private/rules/scala_library.bzl#L42

It took me a good few seconds to understand what you're doing here

Reply:
Done